hedera-mirror-rosetta/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.13-alpine as build
```

Comment:
I think that you should use `golang:1.13` instead of `alpine` as per Rosetta requirements -> https://www.rosetta-api.org/docs/node_deployment.html#ubuntu-image-compatibility

Reply:
+1